docs/contribute.md

```diff
@@ -1,30 +1,40 @@
 ## Contribute
-### Picking up issues to complete in existing commands
-1. Fork repo
-2. Create your own branch
-3. `npm install` inside the command directory
-4. Create changes needed in lambda file
-5. Submit pull request
+### Picking up issues to complete in existing commands and features
+1. Leave a comment on the GitHub issue that you want to work on to be assigned to it
+2. Fork repo
+3. Create your own branch
+4. `npm install` inside the command directory
+5. Create changes needed in lambda file
+6. Run unit tests to make sure everything still passes
+7. Create more unit tests if needed
+8. Submit pull request
 
-### Creating new command
+### Creating new Slack slash command
 1. Fork repo
 2. Create your own branch
 3. Create command directory (kebab-case and using the same name as command)
-4. `npm init` and setup new `package.json`
+4. `npm init` and setup new `package.json`, if needed
 5. Create folder called 'cloudformation' to store CloudFormation template
 6. Create Cloudformation template, the following resources are required:
     * Lambda function
     * Lambda IAM role
     * Lambda permission allowing the SNS to call the handler
-    * SNS (SNS name need to match the command in Slack)
+    * SNS Topic (topic name need to match the command in Slack—e.g. the SNS topic for the `health-tm` slash command needs to also be named `health-tm`)
 7. Create Lambda handler
-8. Submit pull request
+8. Create unit tests
+9. Submit pull request
 
 #### CloudFriend and cfn-config
 MapBox has created [Cloudfriend](https://github.com/mapbox/cloudfriend) and [cfn-config](https://github.com/mapbox/cfn-config) to make it easier to create, deploy, and update CloudFormation templates.
 
+### Creating other Slackbot features
```

Comment:
This is for non-slash commands features (such as the GitHub label-filter or anything similar). Would appreciate feedback on what I wrote here.